src/classes/insightsConnection.ts

```diff
@@ -456,7 +453,10 @@ export class InsightsConnection {
             kdbOutputLog(`User cancelled the Scrathpad execution.`, "WARNING");
```

Comment:
Let's fix the Scratchpad typo here, should also be lowercase "scratchpad" to match other messages